### PR TITLE
MRG, DOC: adds "projectors overview" tutorial

### PR DESCRIPTION
**Pull request by drammock**
Also:
- resolves several TODOs in other tutorials that were waiting for cross-refs to this one
- updates the README of the intro section to better reflect the content of the tutorials in there

### CODE REVIEW

**Comment by drammock** [comment]
rendered tutorial: https://13792-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/plot_projectors_intro.html

File: tutorials/intro/plot_projectors_intro.py [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+# for magnetometers (the middle two, marked "axial"), and two for EEG sensors
+# (the last two, marked "eeg"). We can add them to the :class:`~mne.io.Raw`
+# object using the :meth:`~mne.io.Raw.add_proj` method (note that there is a
+# corresponding method :meth:`~mne.io.Raw.del_proj` that will remove projectors
+# based on their index within the ``projs`` field of ``raw.info``):
+
+raw.add_proj(ecg_projs)
+
+###############################################################################
+# The output says the added projections are "deactivated" because they have
```
Comment (agramfort) [inline tutorials/intro/plot_projectors_intro.py]: glitch here

File: tutorials/intro/plot_projectors_intro.py [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+# projectors can be *delayed* by passing ``proj=False`` to the
+# :class:`~mne.Epochs` constructor. However, even when projectors have not been
+# applied, the :meth:`mne.Epochs.get_data` method will return data *as if the
+# projectors had been applied* (though the :class:`~mne.Epochs` object will be
+# unchanged). Additionally, when performing inverse imaging (i.e., with
+# :func:`~mne.minimum_norm.apply_inverse`), the projectors will be
+# automatically applied. It is also possible to apply projectors manually when
+# working with :class:`~mne.io.Raw`, :class:`~mne.Epochs` or
+# :class:`~mne.Evoked` objects via the object's :meth:`~mne.io.Raw.apply_proj`
+# method.
```
Comment (agramfort) [inline tutorials/intro/plot_projectors_intro.py]: I would remind the del_proj method here

File: tutorials/intro/plot_projectors_intro.py [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file)
+raw.crop(tmax=60).load_data()
+
+###############################################################################
+# In MNE-Python, the environmental noise vectors are computed using `principal
+# component analysis`_, usually abbreviated "PCA", which is why the SSP
+# projectors usually have names like "PCA-v1". The projectors are stored in the
```
Comment (agramfort) [inline tutorials/intro/plot_projectors_intro.py]: I would also mention the acronym SVD as it is common to say that SVD was used for projection
computation.

File: tutorials/intro/plot_projectors_intro.py [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+mags = raw.copy().crop(tmax=2).pick_types(meg='mag')
+mags.plot(butterfly=True, proj=False)
+mags.plot(butterfly=True, proj=True)
+
+###############################################################################
+# Additional ways of visualizing projectors are covered in the tutorial
+# :doc:`../preprocessing/plot_artifacts_correction_ssp`.
+#
+#
+# Loading and saving projectors
```
Comment (jasmainak) [inline tutorials/intro/plot_projectors_intro.py]: there is no section saying that projectors can also be computed if you want?

**Comment by drammock** [comment]
comments addressed. Updated rendering:
https://13814-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/plot_projectors_intro.html

File: tutorials/intro/plot_projectors_intro.py (hunk 4 above)
Comment (drammock) [reply tutorials/intro/plot_projectors_intro.py]: this was added to the first paragraph of the SSP section. The "artifact rejection with SSP" tutorial doesn't (yet) do EOG/ECG artifacts (only the ICA tutorial does), so I've added a TODO to cross-link to that tutorial once I've added those examples to it.

**Comment by agramfort** [comment]
perfect !

thanks heaps @drammock

**Pull request merged**